### PR TITLE
Adds cockroaches

### PR DESCRIPTION
**Pull request by unid15**
My other 2 prs will never get finished edition

fuuck

faaaaaaaaack

(the spammy messages have been removed)
- These creatures from hell crawl on walls and food, and also hide under trash. They can lay eggs under food (and, since their eggs count as food, this means they'll create large nests with **dozens** of eggs)
- Killing them leaves a mess
- Spraying them with toxin will kill them without leaving a mess (only a small roach body)
- They can fly when threatened, gaining speed
- They survive in any environment as long as the temperature is above 0C, and can crawl over computers, grilles and tables. They also survive explosions and nukes
- The infestation event can spawn up to 30 of these disasters

**OTHER CHANGES**
- Acid smoke and acid sprays will now slightly damage simple animals
- Tiny animals (mice, roaches, lizards) no longer produce gibs when butchered

### CODE REVIEW

**Comment by clusterfack** [comment]
Oh god please no, once you get cockroaches in the code you can never get them out

**Comment by ihadtoregisterforthis** [comment]
> They can fly

I hope you are also improving the flamethrower.

Weakness to poison? Weakness to newpapers? Do they transmit viruses? Do they crunch when stepped on?

**Comment by ghost** [comment]
Hitting a cockroach with the goonecode spawning runtime when?

**Comment by unid15** [comment]
Working on it @ihadtoregisterforthis 

haha

Also spraying them with creatine will make them dog-sized and aggressive (_in the future, not in this pr_)

**Comment by Exxion** [comment]
> other 2 prs will never get finished

But you have 3 other PRs

**Comment by Ephx1** [comment]
The only creature I truly hate and you just had to add it.

**Comment by Imagebard** [comment]
Holy shit FUCK those create a huge mess
If I saw that as a janitor in a round I would just space myself or call the shuttle because truly the round has been lost.

**Comment by Shadowmech88** [comment]
> spraying them with creatine will make them dog-sized and aggressive

How strong are they in that form?

**Comment by Exxion** [comment]
> roach meat

**Comment by Imagebard** [comment]
A single chemsprayer could make a thousand aggressive mobs
Do you believe it

I hope you balanced it

File: code/modules/mob/living/simple_animal/roach.dm [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+	turns_since_move = 5 //Remove any delay
+
+	icon_state = "cockroach_fly"
+
+	flying = 1
+	speak_chance = 5
+
+	turns_per_move = 1
+
+	response_help  = "attepts to pet"
```
Comment (KingDairy) [inline code/modules/mob/living/simple_animal/roach.dm]: "attepts" should be "attempts"
am i doing this right? am i allowed to bitch about typos?

**Comment by unid15** [comment]
The creatine stuff isn't done yet, I'll do it after this PR

File: code/modules/mob/living/simple_animal/roach.dm (hunk 1 above)
Comment (unid15) [inline code/modules/mob/living/simple_animal/roach.dm]: :+1:

**Comment by ihadtoregisterforthis** [comment]
Did they... did they all survive to the IED in the gif?

And can they survive the nuke? Petty detail but could break some emulsions otherwise

**Comment by unid15** [comment]
> And can they survive the nuke?

now they can

**Comment by clusterfack** [comment]
I'm not going to lie this is the most horrifying thing I've ever seen, also squash it

**Pull request merged**